CHANGELOG.md

```diff
@@ -5,6 +5,7 @@ Observes [Semantic Versioning](https://semver.org/spec/v2.0.0.html) standard and
 ## [Unreleased]
 ### Added
 - Support for double data type (#72) PR #86
+- Added search functionailty for table list similar to schema. (#73) PR #88
```

Comment:
This line is actually fine but if you look below in line `15`, the issue reference should be `(#81) PR #86`. Missed this before.

Reply:
ah nice catch, sorry my fault